src/jobs/models/taskManager.ts

```diff
@@ -0,0 +1,106 @@
+import { Logger } from '@map-colonies/js-logger';
+import { inject, injectable } from 'tsyringe';
+import { SERVICES } from '../../common/constants';
+
+import { ConnectionManager } from '../../DAL/connectionManager';
+import { EntityNotFound } from '../../common/errors';
+import { TaskRepository } from '../../DAL/repositories/taskRepository';
+import {
+  CreateTasksRequest,
+  CreateTasksResponse,
+  GetTasksResponse,
+  IAllTasksParams,
+  IFindTasksRequest,
+  IGetTaskResponse,
+  IGetTasksStatus,
+  ISpecificTaskParams,
+  IUpdateTaskRequest,
+} from '../../common/dataModels/tasks';
+import { OperationStatus } from '../../common/dataModels/enums';
+import { JobManager } from './jobManager';
+
+@injectable()
+export class TaskManager {
+  private repository?: TaskRepository;
+
+  public constructor(
+    @inject(SERVICES.LOGGER) private readonly logger: Logger,
+    private readonly connectionManager: ConnectionManager,
+    private readonly jobManager: JobManager
+  ) {}
+
+  public async getAllTasks(req: IAllTasksParams): Promise<GetTasksResponse> {
+    const repo = await this.getRepository();
+    const res = await repo.getTasks(req);
+    return res;
+  }
+
+  public async createTask(req: CreateTasksRequest): Promise<CreateTasksResponse> {
+    const repo = await this.getRepository();
+    const jobId = Array.isArray(req) ? req[0].jobId : req.jobId;
+    this.logger.info(`creating task(s) for job ${jobId}`);
```

Comment:
If there is no "debug" log with all parameters - add it

Reply:
move the inner (in repo) debug log to here (first row of method) - this.appLogger.debug(req, 'Create-Task request parameters');
change the info format to the convention and put it after the debug message